src/SplitTrilinearInterpolation.jl

```diff
@@ -0,0 +1,174 @@
+module SplitTrilinearInterpolation
+using DocStringExtensions
+
+import ..Interpolation1D: Linear, Flat, interpolate1d!
+import ..BilinearInterpolation: Bilinear, interpolatebilinear!, get_dims
+
+"""
+    SplitTrilinearHV{VS, VT, HB, W1, W2}
+
+This struct stores the source and target grids and work arrays for split trilinear interpolation.
+
+# Fields
+$(DocStringExtensions.FIELDS)
+"""
+struct SplitTrilinearHV{VS, VT, HB, W1, W2}
+    "Vertical source grid"
+    vsource::VS
+    "Vertical target grid"
+    vtarget::VT
+    "bilinear object containing the horizontal source and target grids"
```

Comment:
```suggestion
    "`Bilinear` object containing the horizontal source and target grids"
```
just to emphasize that this is referring to an object type